handlers/api.go

```diff
@@ -2147,7 +2148,25 @@ func ApiGraffitiwall(w http.ResponseWriter, r *http.Request) {
 	if slotQuery < 10000 {
 		slotQuery = 10000
 	}
-	rows, err := db.ReaderDb.Query("SELECT x, y, color, slot, validator FROM graffitiwall WHERE slot <= $1 AND slot >= $2 ORDER BY slot desc", slotQuery, slotQuery-10000)
+
+	startPxl := uint64(0)
+	endPxl := uint64(999)
+
+	startX := math.MinU64(parseUintWithDefault(r.URL.Query().Get("startx"), startPxl), endPxl)
+	startY := math.MinU64(parseUintWithDefault(r.URL.Query().Get("starty"), startPxl), endPxl)
+	endX := math.MinU64(parseUintWithDefault(r.URL.Query().Get("endx"), endPxl), endPxl)
+	endY := math.MinU64(parseUintWithDefault(r.URL.Query().Get("endy"), endPxl), endPxl)
```

Comment:
Please add a sanity check as to ensure that start values <= end values and return an error if the check fails. Think about adding this info to the swagger comment.

Reply:
As discussed we don't run into any issues for invalid areas since the result will just be empty but it is useful return an error with the info that the area is invalid.